server/index.js

```diff
@@ -2,9 +2,14 @@ const express = require('express');
 
 const app = express();
 const port = 3003;
+const db = require('../data/index.js');
 // const path = require('path');
 // const bodyParser = require('body-parser');
 
 app.use('/', express.static('public'));
 
+app.get('/listings', (req, res) => {
+  db.getAll();
```

Comment:
@VALE46GP getAll doesn't appear to be a method on the object required from (../data/index.js). Is this just a reference to a yet to be implemented method?

Reply:
The object is used for interacting with the db but you're right; it lacked an export statement.